ethical-advertising-works.rst

```diff
@@ -0,0 +1,104 @@
+.. post:: April 5, 2018
+   :tags: advertising, business, sustainability
+
+Ethical Advertising Works
+=========================
+
+It has been two years since we first launched :doc:`ads on Read the Docs <ads-on-read-the-docs>`.
+We figured it was time to report on the results that we've seen,
+and say thanks to those who have helped us along the way.
+
+The Results
+-----------
+
+To put it simply:
+
+**Ethical Advertising works**.
+
+In Q1 2018,
+our estimated revenue from advertising is $75,000.
+This is enough to fund our small team of 4 people to work on Read the Docs full-time.
+
+When we started building our ethical advertising model,
+we didn't know how it would work out.
+We didn't know much about the ad industry,
+and we were doing things different than everyone else.
+
+We are happy to report that you can build advertising that is a win/win/win for all parties:
+
+* Read the Docs wins because we generate revenue, allowing us to sustain our operations
+* Advertisers win because they are able to reach our audience of developers, helping to grow their business.
+* Users win because their privacy is maintained, and they continue to have access to our service.
+
+Tracking users isn't required
+-----------------------------
+
+Along the way we've been trying to prove a simple thesis:
+
+We can have successful advertising without tracking our users.
+So much of the trouble facing the ad industry these days is because of this massive databases of user data.
+
+We have been able to show that we can show ads without knowing exactly who our users are.
+The content they are reading is enough to show them targeted ads,
+without knowing anything about them.
+
+We're now taking a look at what we've learned,
+and how we might help more infrastructure projects reach sustainability with an ethical advertising approach.
+
+Towards an Ethical Ad Network
+-----------------------------
+
+**We have shown that an advertising business model that respects users can also sustain critical open source infrastructure.**
+We hope that we can act as an example for other projects that depend on ads,
```

Comment:
I think this should be a period instead of a comma. I'm not a grammar expert though.